### PR TITLE
Adding test for the generated `serialize` function

### PR DESCRIPTION
**Pull request by oxisto**
Maybe this helps with the missing code coverage

### CODE REVIEW

File: csaf-import/src/test/kotlin/io/github/csaf/sbom/CsafLoaderTest.kt [hunk 1]
```diff
@@ -43,11 +46,18 @@ class CsafLoaderTest {
             result.isSuccess,
             "Failed to \"download\" example-01-aggregator.json from resources."
         )
+
+        var lister = result.getOrNull()
```
Comment (milux) [inline csaf-import/src/test/kotlin/io/github/csaf/sbom/CsafLoaderTest.kt:51]: `val` :wink:

**Review by milux — COMMENTED** [review]
Looks nice, maybe we can do the same for the other Pojos to improve coverage. :thinking:

**Comment by milux** [comment]
The line coverage even goes further down by this, according to `kover`... :see_no_evil:

**Comment by milux** [comment]
Oh well... the only thing that improved was that my custom serializer impl in `csaf-schema` or now "indirectly covered", according to `codecov`. Not exactly a solution for the `@Serializable` issue. How should an annotation be "covered" anyway? What does that even mean? :see_no_evil:

**Comment by oxisto** [comment]
> Oh well... the only thing that improved was that my custom serializer impls in `csaf-schema` are now "indirectly covered", according to `codecov`. Not exactly a solution for the `@Serializable` issue. How should an annotation be "covered" anyway? What does that even mean? 🙈

No idea... this is strange. I would argue that we should exclude the generated files from the code coverage then.

**Review by milux — APPROVED** [review]

**Pull request merged**